src/pepr/zarf.yaml

```diff
@@ -0,0 +1,69 @@
+kind: ZarfPackageConfig
+metadata:
+  name: pepr-uds-core
+  description: 'Pepr Module: A collection of capabilities for UDS Core'
+  url: https://github.com/defenseunicorns/pepr
+  version: 0.5.0
```

Comment:
```suggestion
```

I think we can/should drop this version for simplifying the create/deploy lifecycle.

Reply:
Updated